BotRoyaleUnity/Assets/Scripts/PartHealth.cs

```diff
@@ -66,7 +66,7 @@ public void setRelPos(int x, int z, string type, string direction)
     }
 
     public void subtractDirectionStrength(){
-        if(type != "spike"){
+        if(type == "block"){
```

Comment:
Just wondering what this check is for. Is it that the blocks don't have a direction, so the strength is determined by the block's position relative to the center?